bio/mapdamage2/environment.yaml

```
@@ -4,3 +4,8 @@ channels:
   - defaults
 dependencies:
   - mapdamage2 =2.2.1
+  - r-inline
+  - r-ggplot2
+  - r-gam
+  - r-rcpp
+  - r-rcppgsl
```

From my understanding of the [`mapdamage2` install instructions in its GitHub README](https://github.com/ginolhac/mapDamage#bioconda-installation), this should work:
```suggestion
  - mapdamage2mapdamage2=2.2.1=pyr40_0
```
Although this also seems like something that should be fixable over at bioconda, rather than in this wrapper. But that is out of my depth, so I'll check in with someone more knowledgable.

---

That was how I initially had the environment, but @johanneskoester said it would be better with just the version and not the build string: https://github.com/snakemake/snakemake-wrappers/pull/186#discussion_r494117562

Well, just let me know how do you thing it would be best to have it.

---

Well, usually it's best not to pin too tightly. So optimally, this wrapper would even only pin to `  = mapdamage2 =2.2`. But I am at a loss what exactly is happening here. And both the bioconda package and the snakemake wrapper have passing tests. So I would think that the error you are seeing is something specific to your setup and I can't really help much with debugging there -- you'll have to dig deeper yourself.
And adding dependencies redundantly over here should not change anything in the general setup...

One option to try that sometimes helps, is completely removing your current conda installation and starting from scratch. It's kind of the nuclear option, but is has done the trick for me for some weird situations like this one.